_includes/release-notes/51new-features.md

```diff
@@ -195,6 +195,42 @@ Fixes for inline dialogs and menus have been included to:
 
 ## Premium Plugin changes
 
+### Advcode
+
+The {{site.productname}} 5.1 release includes **Advanced Code** 2.0.2.
+
+**Advanced Code** 2.0.2 has been updated to improve performance on mobile devices.
+
+### Checklist
+The {{site.productname}} 5.1 release includes **Checklist** 1.0.1.
+
+**Checklist** 1.0.1 includes:
+* A fix to allow checklist items to be toggled on an iPad.
+* A fix to ensure the checklist toolbar button is toggled when checklist content is selected.
+
+### Enhanced Media Embed
+
+The {{site.productname}} 5.1 release includes **Enhanced Media Embed** 2.2.4.
+
+**Enhanced Media Embed** 2.2.4 has been updated to include a touch event listener to allow embedded media to execute on mobile platforms.
+
+### Format Painter
+The {{site.productname}} 5.1 release includes **Format Painter** 1.2.0.
+
+**Format Painter** 1.2.0 has been updated to support the mobile platforms. The **Format Painter** will now function as expected for mobile users.
```

Comment:
"the mobile platforms" - should it include "the"?

Reply:
no... no it should not :+1:

Reply:
fixed